src/main/java/org/takes/facets/auth/social/PsGithub.java

```diff
@@ -164,8 +164,16 @@ private String token(final String home, final String code)
             .back()
             .fetch().as(RestResponse.class)
             .assertStatus(HttpURLConnection.HTTP_OK)
-            .as(XmlResponse.class)
-            .xml().xpath("/OAuth/access_token/text()").get(0);
+            .as(XmlResponse.class);
+        String tokenString = null;
+        try {
+            tokenString = xmlres.xml()
+                .xpath("/OAuth/access_token/text()").get(0);
+        } catch (final IndexOutOfBoundsException exce) {
+            throw new HttpException(
+                HttpURLConnection.HTTP_BAD_REQUEST,"Invalid XML",exce);
```

Comment:
@sebing also, let's move the ending `);` to the next line, aligned with the `throw`.